components/grid/export/overview.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Overview
+page_title: Grid - Export Overview
+description: Export basics for the Grid for Blazor.
+slug: grid-export-overview
+tags: telerik, blazor, grid, export
+published: True
+position: 1
+---
+
+# Blazor Grid Export
+
+The Grid for Blazor provides a built-in functionality to export the data to:
+* [CSV](slug:grid-export-csv)
+* [Excel](slug:grid-export-excel)
+* [PDF](slug:grid-export-pdf)
+
+Before proceeding to the dedicated export articles, ensure you are familiar with the following specifics:
+
+## How the Export Works
+
+* If the Grid is using `OnRead` and is exporting all pages, it will fire an additional `OnRead` event at the time of exporting, with a request `PageSize` of `0`. This will enable the component to obtain all data.
+* The time for export will be longer if the Grid has a lot of records and/or in Client-side Blazor (WebAssembly) where all the code runs in the browser and, at the time of writing, is considerably slower than server-side Blazor, and it only has one actual thread. While the file is being generated, the UI will be unresponsive, so you may want to [show a loading sign to the user during the export process](slug:grid-kb-show-loader-while-exporting).
+
+## Requirements
+
+* With Server-side Blazor, the file may become larger than the default SignalR connection limit, and this can disconnect the client and result in an error. Generally, this requires quite a lot of data to happen, but you may need to increase the size limit of the connection in the `ConfigureServices` method of your `Startup.cs` file, for example:
```

Comment:
This does not seem like a requirement to me, but rather a specific way of working or a limitation to be addressed.

Reply:
I've listed this under requirements as the developer must perform the listed step to enable the export of large files to happen. I imagine a limitation would be something that cannot be achieved while here we have a required step for something to happen.